operators/src/plot/histogram.rs

```diff
@@ -972,7 +972,8 @@ mod tests {
                     "type": "OgrSource",
                     "params": {
                         "dataset": {
-                            "internal": dataset_id
+                            "type": "internal",
+                            "dataset": dataset_id
```

Comment:
```suggestion
                            "id": dataset_id
```

Reply:
I think it's more consistent to call the dataset_id `dataset` in the internal and external case

Reply:
You have

```json
{
 "dataset": {
    "dataset":  "name"
  }
}
```